src/Autocomplete/src/AutocompleteResultsExecutor.php

```diff
@@ -71,9 +71,7 @@ public function fetchResults(EntityAutocompleterInterface $autocompleter, string
 
         $results = [];
 
-        if (!method_exists($autocompleter, 'getGroupBy')) {
-            trigger_deprecation('symfony/ux-autocomplete', '2.8', 'Not implementing the "EntityAutocompleterInterface::getGroupBy()" in "%s" is deprecated.', get_debug_type($autocompleter));
-        } elseif (null === $groupBy = $autocompleter->getGroupBy()) {
+        if (null === $groupBy = $autocompleter->getGroupBy()) {
```

Comment:
Still requires a `method_exists()` call, doesn't it?

Reply:
lol, duh